packages/test/test-service-load/src/stressTest.ts

```diff
@@ -143,12 +143,14 @@ export async function stressTest(
 				return new Promise((resolve) => runnerProcess.once("close", resolve));
 			}),
 		);
-	} finally {
 		const endTime = Date.now();
 		console.log(`End time: ${endTime} ms\n`);
 		console.log(`Total run time: ${(endTime - startTime) / 1000}s\n`);
-		await safeExit(0, url);
+	} catch {
+		// Swallow all errors. A previous implementation exited the process here with code 0.
```

Comment:
I agree `safeExit()` in the `finally` block was probably wrong but shouldn't we let errors propagate from here? `main.ts` has a catch that would output them to console. It wasn't getting exercised even if errors happened because of `safeExit()` here, but if something went wrong running the stress test, do we have any way of knowing beside letting errors propagate? It also feels weird that we'd have a clean process exit but no end time and total run time output, those still feel ok in a `finally` block to me.

Reply:
Agree - my original intent with this PR was to avoid making any serious changes to the stressTest while adding the smokeTest (even for the places where it seems fishy), but Tony convinced me to try a PR to clean up exit/error flows a bit first (including allowing this to propagate out).